test/stub/keyring-bridge.js

```diff
@@ -0,0 +1,55 @@
+export const KNOWN_PUBLIC_KEY =
+  '02065bc80d3d12b3688e4ad5ab1e9eda6adf24aec2518bfc21b87c99d4c5077ab0';
+
+export class FakeKeyringBridge {
+  #publicKeyPayload;
+
+  constructor({ publicKeyPayload }) {
+    this.#publicKeyPayload = publicKeyPayload;
+  }
+
+  async init() {
+    return Promise.resolve();
+  }
+
+  async dispose() {
+    return Promise.resolve();
+  }
+
+  async destroy() {
+    return Promise.resolve();
+  }
+
+  updateTransportMethod() {
+    return true;
+  }
+
+  async getPublicKey() {
+    return this.#publicKeyPayload;
+  }
+}
+
+export class FakeTrezorBridge extends FakeKeyringBridge {
```

Comment:
This class can be extended with these methods to test signatures:

* `ethereumSignTransaction`
* `ethereumSignMessage`
* `ethereumSignMessage`

Reply:
Though we can't use the `KNOWN_PUBLIC_KEY` alone to forge signatures, so we'll have to use a root private key instead, to:
* Derive a master public key to be returned in `getPublicKey` (like `KNOWN_PUBLIC_KEY`)
* Derive private keys for each account, usable for signing messages and txs